tests/end_to_end_test.py

```diff
@@ -248,3 +279,30 @@ def test_number_of_module_appearances(ciods, ciod_module_relationship, modules):
     all_module_appearances = [rel for rel in ciod_module_relationship
                               if rel['moduleId'] == module['id']]
     assert len(all_module_appearances) == 2
+
+
+@pytest.mark.endtoend
+class TestUniqueIds:
+    def ids_are_unique(self, dict_list):
+        return len(dict_list) == len(set(d['id'] for d in dict_list))
```

Comment:
I tend to use `set()` because I usually use brackets for dictionaries so it's a little more explicitly defined. As far as I know, as long as the arg to `set()` is an iterable it should work properly. Is there a standard for this?

Reply:
Oh yeah that is totally fine. I agree that the visual similarity between dict and set comps can be kind of confusing and think using `set()` here makes sense. I don't think there's any kind of established standard for this, it's mostly just a stylistic choice.